### DISCUSSION TITLE
Can't open an image

### DISCUSSION THREAD

**Discussion by carelribaltchenko** [ImageMagick/ImageMagick#7025] [Help]
### ImageMagick version

8:6.9.11.60

### Operating system, version and so on

Ubuntu0.22.04.3

### Description

i created an image and i can't open it with imagemagick, when i do this command, display -sample 5000% Image6.mbp it send me this message : 
display-im6.q16: unable to runlength decode image `Image6.bmp' @ error/bmp.c/ReadBMPImage/1020.
but when i try wit anothe image, it works, so I odn't understand why it doesn't work

**Comment by dlemstra** [comment]
You are getting that message because that image is invalid/corrupt.
